domains/neca.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "nekotamogas",
+           "email": "dazzerdev@gmail.com",
+           "discord": "797458672225091594"
+        },
+    
+        "record": {
+            "TXT": "zoho-verification=zb18547320.zmverify.zoho.eu"
```

Comment:
I think you need more records than TXT for zoho (like MX records)

Reply:
It only gave me TXT record and i can choose also between CNAME or something with uploading file on the website.